core/client/app/mixins/ed-editor-api.js

```diff
@@ -129,7 +129,7 @@ var EditorAPI = Ember.Mixin.create({
 
         $textarea.focus();
         // Tell the editor it has changed, as programmatic replacements won't trigger this automatically
-        this.sendAction('onChange');
+        this.set('value', $textarea.val());
```

Comment:
Just found a similar issue here if that provides any further insight - https://github.com/emberjs/ember.js/issues/9543

> I guess that this.$().trigger('change') is the correct incantation over and above this.set('value', $textarea.val()); ?

My preference in this case would be to trigger the `change` event (or possibly `paste` if that's the only one that is intercepted for the `replaceSelection` shortcut) rather than changing the underlying property directly as we're intercepting and preventing the propagation of the DOM event in the shortcuts route. This keeps the behaviour consistent by ensuring that we don't accidentally break anything else that depends on that event firing.